src/Lean/LazyInitExtension.lean

```diff
@@ -1,42 +0,0 @@
-/-
-Copyright (c) 2021 Microsoft Corporation. All rights reserved.
-Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Leonardo de Moura
--/
-prelude
-import Lean.MonadEnv
-
-namespace Lean
-
-structure LazyInitExtension (m : Type → Type) (α : Type) where
```

Comment:
@leodemoura I removed this abstraction as it's now unused and doing lazy initialization *in an elaboration monad* seems like a bad idea in general under parallelism as the elaboration context will not be deterministic. Sounds good?